src/main/webapp/history/history.js

```diff
@@ -19,51 +18,337 @@
                 tooltip.enable();
             });
         });
+		
+		function renderTrendChart(chartDivId, model, settingsDialogId, chartClickedEventHandler) {
+			const chartPlaceHolder = document.getElementById(chartDivId);
```

Comment:
I have these placeholder sections showing up for both trend and distribution:

<img width="185" alt="image" src="https://github.com/jenkinsci/junit-plugin/assets/21194782/e31980fb-1d1d-4c10-aea7-4c511c5c8ca6">


Previously it looked like this:
<img width="96" alt="image" src="https://github.com/jenkinsci/junit-plugin/assets/21194782/18235ed3-6a92-4c78-bd3b-5158c3e176aa">

Reply:
I am not sure how to disable the filled circle. Any tips?

Reply:
There's no direct way, you could either add a CSS rule in history.css to hide `.icon-border` with `display: none !important`  (it would need other CSS rules added as once thats done everything moves up too far.

Or you could add an icon for each, might look nicer, but it would still probably need other changes because currently some icons are overflowing

Reply:
I hid the circle and did more alignment improvements.